services/gateway/sources/version2/mappings/account.js

```diff
@@ -69,7 +65,14 @@ module.exports = {
 			status: 'status,string',
 			rewards: 'rewards,string',
 		},
-		sentVotes: '=',
-		unlocking: '=',
+		sentVotes: ['sentVotes', {
+			delegateAddress: '=,number',
+			amount: '=,number',
+		}],
+		unlocking: ['unlocking', {
+			delegateAddress: '=,number',
+			amount: '=,number',
+			unvoteHeight: '=,number',
```

Comment:
Shouldn't `unvoteHeight` should be an object consists of start and height (as in `services/core/shared/core/compat/sdk_v5/accounts.js`, we are calculating `unvoteHeight` as a object),

```
	unlocking: ['unlocking', {
		amount: '=,string',
		height: {
			start: '=,string',
			end: '=,string',
		},
		delegateAddress: '=,string',
	}],
```
We can clear this with @MichalTuleja ?

Reply:
The change in the mapping is in accordance with the response example in the specification document. In case, it has to be an object we should update the `start` and `end` heights to be instances of `number` instead, for consistency.

```
	unlocking: ['unlocking', {
		amount: '=,string',
		height: {
			start: '=,number',
			end: '=,number',
		},
		delegateAddress: '=,string',
	}],
```

Reply:
Let's wait for @MichalTuleja reply, what he says?

Reply:
According to specification in the issue #250, the property is called `unvoteHeight` and defined as `Number`.

```
      "unlocking": [
        {
          "delegateAddress": "lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu",
          "amount": "string number",
          "unvoteHeight": 30
        }
      ]
```

Do you think the `Number` might be a problem here?

Reply:
I dont think its a problem here as from `lisk-core` we are getting `unvoteHeight` directly.

Reply:
Do you mean we get it as a `Number`? Then we can make it exactly as described above.

Reply:
Yes, we get `unvoteHeight` as a `Number` directly from core.

Reply:
This is exactly what the current change does and is in line with the specifications in #250.

Reply:
As concluded in our last discussion, `unlocking` is now updated to 
```
		unlocking: ['unlocking', {
			delegateAddress: '=,string',
			amount: '=,string',
			height: {
				start: '=,number',
				end: '=,number',
			},
		}],
```